core/utils/dmljob.py

```diff
@@ -63,6 +64,7 @@ def __init__(
         self.omega = None
         self.sigma_omega = None
         self.new_weights = None
+        self.uuid = uuid
```

Comment:
Same as above -doesn't seem like a job is going to have just 1 uuid, it will have a list of mappings.

Reply:
Same as above, jobs should only have 1 uuid (each individual dataset has its own job)